pacifica/policy/status/instrument/by_project_id.py

```diff
@@ -35,8 +35,7 @@ def GET(project_id=None):
         """CherryPy GET method."""
         project_info = InstrumentsByProject._get_instruments_for_project(
             project_id)
-        instruments = {index: info for (
```

Comment:
I'll have to try running the code like you've changed it, but let's just go with it for now.